olp-cpp-sdk-dataservice-read/src/QueryMetadataJob.h

```diff
@@ -34,6 +35,16 @@
 namespace olp {
 namespace dataservice {
 namespace read {
+using EmptyResponse = Response<client::ApiNoResult>;
+
+template <typename QueryType, typename QueryResponseType>
+struct QueryResponse {
+  QueryResponse() = default;
+  explicit QueryResponse(QueryType root) : root_(std::move(root)) {}
+  QueryType root_;
+  EmptyResponse response_;
+  typename QueryResponseType::ResultType list_of_tiles_;
```

Comment:
Structures should not use `_` in the member name as all are public. See https://google.github.io/styleguide/cppguide.html#Variable_Names